client/app/types/course/statistics/assessmentStatistics.ts

```diff
@@ -12,7 +12,7 @@ interface AssessmentInfo {
 interface MainAssessmentInfo extends AssessmentInfo {
   isAutograded: boolean;
   questionCount: number;
-  isLiveFeedbackEnabled: boolean;
+  liveFeedbackEnabled: boolean;
```

Comment:
I got it that the `isLiveFeedbackEnabled` is changed here, but actually it should be done in the earlier commit (on the same one as when you change the one in jbuilder). Please change it over there instead of here

Reply:
done